### ISSUE TITLE
Wrong Javascript load order when p:inputTextarea is the first Primefaces component on a page

### ISSUE THREAD

**Issue by laurinmueller** [primefaces/primefaces#2246]
## 1) Environment
- PrimeFaces version: 6.0
- Does it work on the newest released PrimeFaces version? Version? No
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) I don't know
- Application server + version: Websphere 8.5.5.8
- Affected browsers: all

## 2) Expected behavior
no javascript error in the browser developer console

## 3) Actual behavior
Javascript error: 
```
Uncaught ReferenceError: PrimeFaces is not defined
    at components.js.xhtml:1
```
this happens because of the loading order of the core and the components javascript.

![image](https://cloud.githubusercontent.com/assets/1231069/24761906/1145e19a-1aed-11e7-84bf-f2b90b1e48be.png)

when i add a <p:inputText....> before the textarea, the core.js will be loaded before the components.js and everything is fine.

## 4) Steps to reproduce
try to open a page with a textarea as the first primefaces component

## 5) Sample XHTML
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://java.sun.com/jsf/html"
      xmlns:f="http://java.sun.com/jsf/core"
      xmlns:ui="http://java.sun.com/jsf/facelets"
      xmlns:p="http://primefaces.org/ui">


        <h:head>
            <title>Test</title>
        </h:head>

        <h:body id="pgBody">
		<h:form>               		   
		    <p:inputTextarea rows="10" cols="50"  />
		</h:form>

  		</h:body>

    

</html>
```
## 6) Sample bean

not needed

**Comment by tandraschko** [comment]
already fixed by #1573

**Comment by laurinmueller** [comment]
Thanks!
